### DISCUSSION TITLE
error: operator '>' has no left operand when compiling for nrf52840

### DISCUSSION THREAD

**Discussion by kimgalibert** [shanteacontrols/OpenDeck#225] [General]
i'm trying to build for the nfr52840 but i'm getting this error:

>  Building C object CMakeFiles/usb.dir/modules/core/modules/tinyusb/src/device/usbd.c.obj
> In file included from /home/kim/OpenDeck/modules/core/modules/tinyusb/src/device/usbd.c:27:
> /home/kim/OpenDeck/modules/core/scripts/../modules/tinyusb/src/tusb_option.h:431:28: error: operator '>' has no left operand
>   431 | #if CFG_TUD_ENDPOINT0_SIZE > 64
>       |                                                       ^

there are a couple of those and then this one as well:

> 
> /home/kim/OpenDeck/modules/core/modules/tinyusb/src/device/usbd.c:987:53: error: expected expression before ';' token
>   987 |         mod_request.wLength = CFG_TUD_ENDPOINT0_SIZE;
>       |                                                                                               ^
> 

any help would be greatly appreciated, thanks.

**Comment by paradajz** [comment]
Which operating system?
